doc/libpmemblk/pmemblk_ctl_get.3.md

```diff
@@ -95,6 +95,13 @@ impact of pagefaults. Affects only the _UW(pmemblk_open) function.
 
 Always returns 0.
 
+sds.at_create | rw | global | int | int | - | boolean
+
+If set, force-enables or force-disables SDS feature during pool creation.
```

Comment:
This is not helpful enough if user doesn't know what SDS is. You should reference man page that explains that.